src/source/ObjcppMarshal.scala

```diff
@@ -34,9 +34,7 @@ class ObjcppMarshal(spec: Spec) extends Marshal(spec) {
     case o: MOpaque =>
       List(ImportRef(q(spec.objcBaseLibIncludePrefix + "DJIMarshal+Private.h")))
     case d: MDef => d.defType match {
-      case DEnum =>
-        List(ImportRef(q(spec.objcBaseLibIncludePrefix + "DJIMarshal+Private.h")))
```

Comment:
I couldn't just add the C++ header to the list in references() because of the need to return only a single header file name in include() below.  That's called from yaml generation, in a spot which seems to expect only a single header file, not a list of them.